doc/api/zlib.md

```diff
@@ -301,6 +301,7 @@ ignored by the decompression classes.
 * `strategy` {integer} (compression only)
 * `dictionary` {Buffer|TypedArray|DataView} (deflate/inflate only, empty dictionary by
   default)
+* `info` {boolean} (If `true`, returns an object with `buffer` and `engine`)
```

Comment:
nit: I’d drop the parentheses, and maybe add `properties` at the end to be explicit